### DISCUSSION TITLE
yes more and more ... but need a list

### DISCUSSION THREAD

**Discussion by RamazanGeven** [500Foods/TMS-XData-TemplateDemoData#1] [Ideas]
Can we think about to use sphinx for auth ?

**Comment by 500Foods** [comment]
Sure.  Also, just using TWebAuth as was demonstrated by Bruno during the recent TMS WEB Core 2.1 presentation.  I've not done anything fancy with authentication because I wanted to be sure that it worked first with XData.  That seems to be sorted out now, so adding other mechanisms is certainly something we can try and implement.
